ndb/src/google/cloud/ndb/query.py

```diff
@@ -907,7 +907,7 @@ def __init__(self, kind=None, ancestor=None, filters=None, orders=None,
             else:
                 if not isinstance(ancestor, model.Key):
                     raise TypeError("ancestor must be a Key; "
-                                    F"received {ancestor}")
+                                    "received {ancestor}")
```

Comment:
The `F` should be moved to the start of the literal (before `"ancestor`) so that the value actually gets formatted.  Likewise for all of these changes.